en/canary-deployment-tidb-operator.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Canary Deployment of TiDB Operator
+summary: Learn how to canary upgrade TiDB Operator and how to deploy multiple TiDB Operator to manage different TiDB clusters.
+---
+
+# Canary Deployment of TiDB Operator
+
+This document describes how to canary upgrade TiDB Operator and how to deploy multiple TiDB Operator to manage different TiDB clusters.
+
+## Scenarios
+
+- Canary upgrade TiDB Operator.
+- Deploy multiple TiDB Operator to manage different TiDB clusters.
```

Comment:
Can we say `multiple sets of TiDB Operator` or `multiple TiDB Operator sets` instead of `multiple TiDB Operator`?

Reply:
Thanks! I've been troubled by the plural forms for a while ... `multiple sets of TiDB Operator` SGTM

Reply:
fixed in a832dba